config/ant-phase-verify.xml

```diff
@@ -138,7 +140,9 @@
                 >
       <path>
         <fileset dir="src"
-            includes="it/resources/**/*,test/resources/**/*,test/resources-noncompilable/**/*">
+            includes="it/resources/**/*,test/resources/**/*,test/resources-noncompilable/**/*
+            it/resources-noncompilable/**/*">
```

Comment:
can you explain missed `,` between paths?

Reply:
Done